x-pack/plugins/spaces/server/config.test.ts

```diff
@@ -36,10 +36,10 @@ describe('spaces config', () => {
         const { messages, migrated } = applyConfigDeprecations({ ...originalConfig });
 
         expect(messages).toMatchInlineSnapshot(`
-        Array [
-          "Disabling the Spaces plugin (xpack.spaces.enabled) will not be supported in the next major version (8.0)",
-        ]
-      `);
+          Array [
+            "Disabling the Spaces plugin (xpack.spaces.enabled) will not be supported in the next major version (8.0).",
```

Comment:
Can we lead with the unsupported version and the user action? For example:

`In 8.0.0 and later, using `xpack.spaces.enabled` to disable the Spaces plugin is unsupported.`

Reply:
Sure, no problem 👍 I'll change it like so:

```suggestion
            "In 8.0.0 and later, using \\"xpack.spaces.enabled\\" to disable the Spaces plugin is unsupported.",
```